### PR TITLE
fix test target property #844

### PR DESCRIPTION
**Pull request by btkobayashirun**
Please review #844 .

### CODE REVIEW

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java [hunk 1]
```diff
@@ -402,7 +402,7 @@ public void test06_Query_Display() {
 
         // output 06_08 Test
         assertThat(driver.findElement(By.xpath(
-                "//a[contains(@href, '?page=1&size=10&age=0&countries=&_dateOfBirth=&main=false&name=TEST%5B%5D%23%2B%3D%26TEST')]"))
+                "//a[contains(@href, '?page=1&size=10&age=0&countries=&_dateOfBirth=&main=&name=TEST%5B%5D%23%2B%3D%26TEST')]"))
                 .getText(), is("2"));
     }
```
Comment (yoshikawaa) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java]: - [ ] It is better to have 3 patterns of `main=true` / `main=false` / `main=`.
It is especially important that `false` / `null` are bound as intended.

Comment (btkobayashirun) [reply terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java]: added Boolean false pattern to test case 06_03.

3b696f16a7f6d722295abfb194bb9c212216936a

**Comment by btkobayashirun** [comment]
Added test pattern to bind Boolean to form.

9ae93a089b22d1b42d3d649f4bbcaf427b1d099b

**Comment by btkobayashirun** [comment]
Modified to include Booleans in nested JavaBeans.

2be3e08c0006f829607fdb271fec460e84e65923

**Pull request merged**